### ISSUE TITLE
finishTransactionIOS/finishTransaction this is not working in IOS 

### ISSUE THREAD

**Issue by Mitdd9707** [dooboolab/react-native-iap#1019]
### Version of react-native-iap
4.4.8
### Version of react-native
0.61.5
### Platforms you faced the error (IOS or Android or both?)
IOS
### Expected behavior
I click in the purchase button and I getting the error is undefined finishTransactionIOS method give me an error (undefined/null).
I send finishtransaction this Purchase Object
I send finishTransactionIOS this purchase.transactionId 
all ready send but response was (undefined/null).
<img width="425" alt="Screenshot 2020-05-27 at 6 32 23 PM" src="https://user-images.githubusercontent.com/64424107/83021971-72109400-a048-11ea-9407-6fdb6515b53c.png">

### Tested environment (Emulator? Real Device?)
Real Device (I am Using IphoneX)
### Steps to reproduce the behavior
@hyochan **Can you please help me?**

**Comment by anastely** [comment]
Can you share your code snippet?
I guess I've faced something like this, and want to be sure if it related or not on my issue #1020

**Comment by tomerdev** [comment]
It looks like you didn't use a promise?
You should use await when calling finishTransaction

**Comment by hyochan** [comment]
@Mitdd9707 Can this be closed? I think you should share some of your code.

**Comment by bcbcbcbcbcl** [comment]
I am having the same issue, this is the code I used. 
```
      purchaseUpdateSubscription = RNIap.purchaseUpdatedListener(
        async (purchase) => {
          console.log(purchase);
          const receipt = purchase.transactionReceipt;
          if (receipt) {
            try {
              const ackResult = await RNIap.finishTransaction(purchase);
              console.log('ackResult', ackResult);
            } catch (ackErr) {
              console.warn('ackErr', ackErr);
            }
  
            Alert.alert('purchase error', JSON.stringify(receipt));
          }
        },
      );
```

**Comment by n0umankhan** [comment]
+ i'm having the same issue

**Comment by jkz27** [comment]
Do we have an update on this???

**Comment by hyochan** [comment]
Closes via #1160
